package.json

```diff
@@ -24,7 +24,7 @@
     "browserfs": "^1.4.3",
     "core-js": "^3.27.2",
     "isomorphic-git": "^1.21.0",
-    "leto-modelizer-plugin-core": "github:ditrit/leto-modelizer-plugin-core#0.13.0",
+    "leto-modelizer-plugin-core": "github:ditrit/leto-modelizer-plugin-core#0.15.0",
```

Comment:
Remove this line and put the pr in draft until @justinehell have done his pull request :)

Reply:
Done